examples/swagger/swagger/public/definitions/InterestedParty.yaml

```diff
@@ -0,0 +1,11 @@
+allOf:
+  - $ref: 'InvolvedParty.yaml'
```

Comment:
Should we consider a different example then one that is Insurance focused, just so we're not bleeding over any knowledge-share from client work?